packages/assets/css/src/scss/_components/_spinner/_modifiers/_edit-contained.scss

```diff
@@ -0,0 +1,23 @@
+@use "sass:math";
+@use "../../../_utils/utils" as *;
+
+/// Build contained
+///
+/// @type modifier
+/// @author WPMU DEV
+///
+/// @param {String} $block - Main block name
+@mixin edit-contained($block) {
+	// DIR: Left to right.
+	// THEME: None.
+	@include sui-class($rtl: false, $theme: null) {
+		@include block($block) {
+			@include modifier(contained) {
```

Comment:
@abder Why did we introduce this variation?

Reply:
In many pages in THC, ( and I assume it will be the same for all plugins as they have similar page layouts ) we have a spinner that's centered vertically and horizontally in a contained of that size, this variation will be generally used for a page when it's loading

Reply:
@abder But the current spinner that we have in storybook has position absolute and it's left, top, bottom, right 0 set so If the outer container height is 200px it will align center to that.

Reply:
@abder That's when we use the spinner hook, not the case for the regular Spinner component

Reply:
okay